Changed the reuses title to Built with OTA

emmanuellar opened the pull request.

MattiSG reviewed (approved):
Thanks @emmanuellar!
I added the associated French translation, and rewrote the subtitle so “Open Terms Archive” and “to build” are not repeated 🙂

emmanuellar commented:
> Thanks @emmanuellar! I added the associated French translation, and rewrote the subtitle so “Open Terms Archive” and “to build” are not repeated 🙂

Thank you @MattiSG 🙌
